engine/src/main/java/org/terasology/engine/logic/console/ConsoleImpl.java

```diff
@@ -135,7 +139,7 @@ public void addMessage(String message, MessageType type, boolean newLine) {
     @Override
     public void addMessage(Message message) {
         String uncoloredText = FontUnderline.strip(FontColor.stripColor(message.getMessage()));
-        logger.info("[{}] {}", message.getType(), uncoloredText);
+        logger.atInfo().addArgument(() -> message.getType()).addArgument(() -> uncoloredText).log("[{}] {}");
```

Comment:
simple getter method, so impact neglectable
```suggestion
        logger.info("[{}] {}", message.getType(), uncoloredText); //NOPMD
```